bin/scripts/lib/fonts.json

```diff
@@ -462,6 +462,17 @@
       "repoRelease": true,
       "description": "Modern with ligatures"
     },
+    {
+      "unpatchedName": "Maple Mono",
+      "RFN": false,
+      "patchedName": "MapleMono",
+      "folderName": "MapleMono",
+      "imagePreviewFont": "MapleMono Nerd Font",
+      "linkPreviewFont": "maple",
+      "caskName": "maple",
```

Comment:
There is an existing `font-maple` Cask, that does contain only some few fonts (see below).
I guess THIS PR is about the `SC` version of Maple, and should probably have `sc` in the Cask name?

```
cask "font-maple" do
  version "6.3"
  sha256 "029e0ec0ffd0185cfdfb19f5dab7a489ad7fa7047166fbfe1fe6666625dcc5c6"

  url "https://github.com/subframe7536/Maple-font/releases/download/v#{version}/MapleMono.zip"
  name "Maple Mono"
  desc "Nerd Font font with round corners"
  homepage "https://github.com/subframe7536/Maple-font"

  livecheck do
    url :url
    strategy :github_latest
  end

  font "ttf/MapleMono-Bold.ttf"
  font "ttf/MapleMono-BoldItalic.ttf"
  font "ttf/MapleMono-Italic.ttf"
  font "ttf/MapleMono-Regular.ttf"

  # No zap stanza required
end
```

Reply:
Hmm the Cask is not updated (i.e. it is 6.3), so probably this is created by a 3rd party.
Maybe upstream want to amend the Cask. I could help them if they need help.

```console
$ git log --pretty=format:"%h%x09%an%x09%ad%x09%s" font-maple.rb
bac691e1d       miccal          Mon Jul  3 17:46:48 2023 +0800   Various: add zap comment
74dd198ff       Ocean Yang      Thu Apr 20 23:34:33 2023 +0800  font-maple 6.3 (#7408)
889f93e7f       Ocean Yang      Sat Feb 25 14:40:04 2023 +0800  Update font-maple from 6.1 to 6.2 (#7152)
eb2e7f7fc       Bevan Kay       Tue Feb 14 03:10:32 2023 +1100  Update font-maple from 5.5 to 6.1 (#7123)
248224f07       Ryan            Fri Nov 11 23:08:30 2022 -0500  font-maple: Update name (#6599)
cd35093ab       Bevan Kay       Mon Oct 31 11:01:30 2022 +1100  Update font-maple from 5.0 to 5.5 (#6560)
a8e6cf6d9       Cheng Yao       Wed Aug 24 10:38:53 2022 +0800  Create font-maple.rb (#6156)
```